.eslintrc

```diff
@@ -5,14 +5,20 @@
     "node": true
   },
   "parser": "@typescript-eslint/parser",
-  "plugins": ["import", "react-hooks", "testing-library", "@chanzuckerberg/stories"],
+  "plugins": [
+    "import",
+    "jsx-a11y",
+    "react-hooks",
+    "testing-library",
+    "@chanzuckerberg/stories"
+  ],
   "extends": [
-    "eslint:recommended",
-    "plugin:@chanzuckerberg/stories/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:prettier/recommended",
```

Comment:
curious, is this better than `eslint:recommended`? noticed you removed that one

Reply:
I'm not entirely sure why this line is changed. Lookin now.